src/dftbp/math/lapackroutines.F90

```diff
@@ -323,6 +324,27 @@ subroutine gesv_dcomplex(aa, bb, nEquation, nSolution, iError)
   end subroutine gesv_dcomplex
 
 
+  !> Double precision version of gesv
```

Comment:
Comment doesn't match routine, also doesn't seem to have coverage according to github.

Reply:
Sorry, I forgot to remove the routine I added to solve the (at that time thought to be complex-valued) linear system of the complex Broyden.